gadget/layout.go

```diff
@@ -39,6 +45,12 @@ type LayoutConstraints struct {
 	// SkipResolveContent will skip resolving content paths
 	// and `$kernel:` style references
 	SkipResolveContent bool
+
+	// SkipLayoutStructureContent will skip laying out
+	// content structure data to the volume. This is used when
+	// only the partitions need to get created and content gets
+	// written later.
+	SkipLayoutStructureContent bool
```

Comment:
should this imply the other one (SkipResolveContent) ? do we need to do the other one if we are not doing this? maybe they should be turned into some sort of level of resolving value instead?

Reply:
TBH I was mostly using this very simple separation to avoid a larger refactor but I'm happy to look more into making this a "levels" thing. It was really about expedience because with this the changes inside gadget are minimal.

Reply:
(I could add a TODO as well to revisit this?)

Reply:
In any cases, added a test.

Reply:
I changed this now to have a new IgnoreContent that wil imply the other one.